app/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  root: true,
+  extends: '@react-native-community',
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        '@typescript-eslint/no-shadow': ['error'],
+        'no-shadow': 'off',
+        'no-undef': 'off',
```

Comment:
Is this just default or what you're proposing? I think we need to add more rules. Probably not in this PR but wanted to know your intention.

Reply:
This file is entirely defaults, let's change things up as we see fit. 👍